src/lib.rs

```diff
@@ -339,27 +339,13 @@ pub fn partial_ge<T: PartialOrd>(a: &T, b: &T) -> bool {
 /// Return the minimum of `a` and `b` if they are comparable.
 #[inline]
 pub fn partial_min<'a, T: PartialOrd>(a: &'a T, b: &'a T) -> Option<&'a T> {
-    if let Some(ord) = a.partial_cmp(b) {
-        match ord {
-            Ordering::Greater => Some(b),
-            _ => Some(a),
-        }
-    } else {
-        None
-    }
+    a.partial_cmp(b).map(|ord| if ord.is_ge() { b } else { a })
```

Comment:
To avoid a change in behavior, this should be `is_gt` (and similar below)

Reply:
`is_ge` == `matches!(self, Greater)` which is what was there before.

Reply:
That's incorrect. The "e" stands for "equals", as in "greater or equals". This is illustrated in [the documentation](https://doc.rust-lang.org/std/cmp/enum.Ordering.html#method.is_ge).